tartufo/commands/scan_remote_repo.py

```diff
@@ -53,10 +53,11 @@ def main(
     include_submodules: bool,
 ) -> GitRepoScanner:
     """Automatically clone and scan a remote git repository."""
+    target_branch = f"origin/{branch}" if branch else None
```

Comment:
This might potentially re-introduce #203. Since we control the clone, can we be sure to specify the remote name as `origin` somehow? Or instead of hard-coding `origin` here, can we inspect the repository extract the remote name?

Reply:
Our clone process always names the remote source "origin" (I checked). If the underlying library changes, that could be a problem. I'll look at putting in a step to retrieve the remote name just in case.

Reply:
My concern is, `GitPython` uses actual `git` commands under the surface, which ends up reading user config... so could a user have their config set up such that it automatically clones with a different name?

Reply:
I just added a commit that returns the actual remote source when the repository is cloned, and then uses that.